### PR TITLE
fix bot

### PR DESCRIPTION
**Pull request by muarachmann**
@ivange94 this should give. Lets give it a try and see :)

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -39,7 +39,7 @@ def check():
                 sc.api_call(
                     "chat.postMessage",
                     channel=general_channel,
-                    text="Hi @" + user["profile"]["display_name"] + ", \n" + welcome_template
+                    text="Hi <@"+user["id"]+"> ," + welcome_template
```
Comment (ivange94) [inline app.py]: This won't work. Get the username with `events_data["event"]["user"]`

Comment (muarachmann) [reply app.py]: @ivange94 we have the user object here `user = events_data["event"]["user"]` and grabbing the `id` will just be that `user['id']`. it should return something like `UA7349S` for example. And to reference someone from the docs is a `<@id>` where id is the user's id

Comment (ivange94) [reply app.py]: it won't work. I am saying so because I tested this.

Comment (ivange94) [reply app.py]: ` events_data["event"]["user"]` is enough

Comment (ivange94) [reply app.py]: Part of the json that is been sent. "user" is a string which represents the users display_name and an object.
```json
"event": {
                "type": "name_of_event",
                "event_ts": "1234567890.123456",
                "user": "UXXXXXXX1",
                ...
        },
...
```

**Pull request merged**